docs/Basic-Guide.md

```diff
@@ -134,6 +86,7 @@ If the learn.py runs correctly and starts training, you should see something lik
 ![Training running](images/training-running.png)
 
 You can press Ctrl+C to stop the training, and your trained model will be at `ml-agents/python/models/<run-identifier>/<env_name>_<run-identifier>.bytes`, which corresponds to your model's latest checkpoint. You can now embed this trained model into your internal brain by following the steps below, which is similar to the steps described [above](#play-an-example-environment-using-pretrained-model). 
```

Comment:
After your change of in-editor traning, the trained model is no longer at this place, also it is no longer in this format.

Reply:
Sorry, just saw the note section after this. This part for the training after the build of executable should be in the "Learning-Environment-Executable.md" page.